packages/block-library/src/social-links/editor.scss

```
@@ -98,3 +97,23 @@
 .wp-social-link.wp-social-link__is-incomplete:focus {
 	opacity: 1;
 }
+
+.wp-block-social-links .block-list-appender {
+	position: static; // display inline.
+
+	.block-editor-button-block-appender.components-button.components-button {
+		padding: $grid-unit-10;
+	}
+}
+
+.wp-block-social-links {
+	&.has-small-icon-size .block-editor-button-block-appender.components-button.components-button {
+		padding: $grid-unit-10 / 2;
```

My editor is giving a warning for this:
```
Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
[1] [0] 
[1] [0] Recommendation: math.div($grid-unit-10, 2)
[1] [0] 
[1] [0] More info and automated migrator: https://sass-lang.com/d/slash-div
```suggestion
		padding: math.div($grid-unit-10, 2);
```
```

---

TIL. I'll fix it.